src/main/java/org/fcrepo/sword/http/SWORDServiceResource.java

```diff
@@ -28,15 +31,25 @@
  */
 @Scope("request")
```

Comment:
Again, I went with the OAI Provider example here.  But `request` isn't actually needed. I checked the Spring MVC documentation on scopes again and there is no reason for this service not to be a singleton. This [StackOverflow question](http://stackoverflow.com/questions/16795303/must-spring-mvc-classes-be-thread-safe) makes it pretty clear.

Why `@Scope("request")` isn't working still remains a question though.